components/tyk-mdcb/values.yaml

```
@@ -0,0 +1,280 @@
+  # - MDCBLicense - Sets MDCB license key
+  useSecretName: ""
+
+  # Tyk MDCB license
+  # It is used to set TYK_MDCB_LICENSE
+  license: ""
+
+  # The rpc port which worker gateways will connect to.
+  # This opens a port on MDCB container and MDCB service targets this port.
+  # It is used to set TYK_MDCB_LISTENPORT
+  listenPort: 9090
+
+  # Set to true if you are using a hashed configuration installation of Tyk, otherwise set to false.
+  # It is used to set TYK_MDCB_HASHKEYS
+  hashKeys: true
+
+  # Allows usage of self-signed certificates when connecting to an encrypted Redis database.
+  # It is used to set TYK_MDCB_STORAGE_REDISSSLINSECURESKIPVERIFY
+  redisSSLInsecureSkipVerify: false
+
+  # When it is set to true, instead of sending analytics directly to MongoDB,
+  # MDCB can send analytics to Redis. This will allow tyk-pump to pull
+  # analytics from Redis and send to your own data sinks.
+  # It is used to set TYK_MDCB_FORWARDANALYTICSTOPUMP
+  forwardAnalyticsToPump: true
+
+  probes:
+    # This port lets MDCB allow standard health checks.
+    # It also defines the path for liveness and readiness probes.
+    # It is used to set TYK_MDCB_HEALTHCHECKPORT
+    healthCheckPort: 8181
+    # liveness includes details about liveness probe used in MDCB Deployment.
+    liveness:
+      # path represents the http path to be used in liveness probe in MDBC deployment.
+      path: "/health"
+      # HTTP represents the scheme to be used in liveness probe in MDBC deployment.
+      scheme: "HTTP"
+      # initialDelaySeconds specifies the seconds used in liveness probe as initial delay.
+      initialDelaySeconds: 5
+      # The periodSeconds specifies the duration in seconds to perform a liveness probe.
+      periodSeconds: 2
+      # timeoutSeconds represents the number of seconds after which the probe times out.
+      timeoutSeconds: 3
+      # failureThreshold represents the consecutive number of failures in a row for Kubernetes to consider that the overall check has failed.
+      failureThreshold: 2
+    # readiness includes details about readiness probe used in MDCB Deployment.
+    readiness:
+      # path represents the http path to be used in readiness probe in MDBC deployment.
+      path: "/health"
+      # HTTP represents the scheme to be used in readiness probe in MDBC deployment.
+      scheme: "HTTP"
+      # initialDelaySeconds specifies the seconds used in readiness probe as initial delay.
+      initialDelaySeconds: 1
+      # The periodSeconds specifies the duration in seconds to perform a readiness probe.
+      periodSeconds: 10
+      # timeoutSeconds represents the number of seconds after which the probe times out.
+      timeoutSeconds: 3
+      # failureThreshold represents the consecutive number of failures in a row for Kubernetes to consider that the overall check has failed.
+      failureThreshold: 3
+
+  # replicaCount specifies number of replicas to be created if kind is Deployment.
```

```suggestion
  # replicaCount specifies number of replicas to be created.
```
We support deployment kind only